server/http.go

```diff
@@ -61,7 +61,7 @@ func (p *Plugin) handleWebhook(w http.ResponseWriter, r *http.Request) {
 }
 
 func (p *Plugin) handleStandardWebhook(w http.ResponseWriter, r *http.Request, webhook *zoom.Webhook) {
-	if webhook.Status != zoom.WebhookStatusStarted {
+	if webhook.Status != zoom.WebhookStatusEnded {
```

Comment:
What? Why did this only recently become a problem?

Reply:
I'm not sure, I believe because it was recently upgraded in community server to version 1.1.1? @levb